### ISSUE TITLE
Do I need to shuffle the order of image path in annotation file?

### ISSUE THREAD

**Issue by psychAo** [open-mmlab/mmclassification#892]
### Checklist

- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

I use my own dataset for training, say a total of 3 classes, so I follow the tutorial to organize my annotation file `train.txt` like this:
```
train/cls0img1.jpg 0
train/cls0img2.jpg 0
train/cls0img3.jpg 0
train/cls1img1.jpg 1
train/cls1img2.jpg 1
train/cls1img3.jpg 1
train/cls2img1.jpg 2
train/cls2img2.jpg 2
train/cls2img3.jpg 2

```
So do I need to shuffle the above order like this, e.g.:
```
train/cls2img2.jpg 2
train/cls0img2.jpg 0
train/cls1img2.jpg 1
train/cls0img1.jpg 0
train/cls1img3.jpg 1
train/cls0img3.jpg 0
train/cls2img3.jpg 2
train/cls1img1.jpg 1
train/cls2img1.jpg 2

```
Because if you don't shuffle the order of feeded images when training a neural network, the results will be poor.
The `shuffle` in PyTorch's `data_loader` only disrupts the data in one batch, and does not shuffle the entire dataset.
All in all, do i have to shuffle the path of images in annotation file `train.txt` by myself, or mmcls already done this for me (If so, where in the code of mmcls?)

**Comment by Ezra-Yu** [comment]
No, we shuffle the dataset by default when building dataloader, refer to https://github.com/open-mmlab/mmclassification/blob/d2e505415040bf5329ab218bb6fe3d899f176cd5/mmcls/datasets/builder.py#L65
